manifest-build-tools/requirements.txt

```diff
@@ -0,0 +1,3 @@
+subprocess32==3.2.7
```

Comment:
Any reason to use a specific version instead of latest ?
by the way, subprocess32 requires "apt-get install python-dev" first. I setup a PT story for this . we can revisit it later.